jenkins-design-language/src/js/components/material-ui/SvgIcon/SvgIcon.js

```diff
@@ -66,6 +70,7 @@ class SvgIcon extends Component {
     return (
       <svg
         {...other}
+        focusable={focusable}
```

Comment:
Why not just drop the option and have `focusable={false}`? ... but wasn't this change already done? (as @sophistifunk said)